lib/these.js

```diff
@@ -83,23 +88,48 @@ These.prototype.map = function (f) {
     return new These(this._this, f(this._that));
 };
 
+This.prototype[FL.map] = This.prototype.map;
+That.prototype[FL.map] = That.prototype.map;
+These.prototype[FL.map] = These.prototype.map;
+
 // Apply : ap
 
 This.prototype.ap = function (x) { return x.map(this._this); };
 That.prototype.ap = function (x) { return x.map(this._that); };
 These.prototype.ap = function (x) { return x.map(this._that); };
```

Comment:
I just realized that the `fantasy-check` which was introduced in #1 by @stoeffel has not been updated in quite some time - https://github.com/fantasyland/fantasy-check - I suppose that does not reflect the 3. fantasy-land spec